README.md

```diff
@@ -1,39 +1,134 @@
-# pd 
+# PD 
 
 [![Build Status](https://travis-ci.org/pingcap/pd.svg?branch=master)](https://travis-ci.org/pingcap/pd)
 [![Go Report Card](https://goreportcard.com/badge/github.com/pingcap/pd)](https://goreportcard.com/report/github.com/pingcap/pd)
 
-Pd is the abbreviate for placement driver.
+PD is the abbreviation for Placement Driver. It is used to manage and schedule the [TiKV](https://github.com/pingcap/tikv) cluster. 
+
+PD supports distribution and fault-tolerance by embedding [etcd](https://github.com/coreos/etcd). 
+
+## Build
+
+1. Make sure [​*Go*​](https://golang.org/) (version 1.5+) is installed.
+2. Use `make build` to install PD. PD is installed in the `bin` directory. 
 
 ## Usage
 
-+ Install [*Go*](https://golang.org/), (version 1.5+ is required).
-+ `make build`
-+ `./bin/pd-server --etcd=127.0.0.1:2379`
+### PD ports
+
+You can use the following default ports in PD:
 
-## Docker
++ **1234**: for client requests with customized protocol.
++ **9090**: for client requests with HTTP.
++ **2379**: for embedded etcd client requests 
++ **2380**: for embedded etcd peer communication.
+
+You can change these ports when starting PD.
+
+### Single Node
 
-### Build
+```bash
+# Set correct HostIP here. 
+export HostIP="192.168.199.105"
 
+pd-server --cluster-id=1 \
+          --addr=0.0.0.0:1234 \
+          --advertise-addr="${HostIP}:1234" \
+          --http-addr="0.0.0.0:9090" \
+          --etcd-name="default" \
+          --etcd-data-dir="default.pd" \
+          --etcd-listen-peer-url="http://0.0.0.0:2380" \
+          --etcd-advertise-peer-url="http://${HostIP}:2380" \
+          --etcd-listen-client-url="http://0.0.0.0:2379" \
+          --etcd-advertise-client-url="http://${HostIP}:2379" \
+          --etcd-initial-cluster="default=http://${HostIP}:2380" \
+          --etcd-initial-cluster-state="new"  
 ```
-docker build -t pingcap/pd .
+
+Command flag explanation:
```

Comment:
Please add "the" before all the explanations and make the first capitalized letter uncapitalized.